numpy/core/src/multiarray/lowlevel_strided_loops.c.src

```diff
@@ -267,11 +286,16 @@ _strided_to_strided(char *dst, npy_intp dst_stride,
 }
 
 static int
-_swap_strided_to_strided(char *dst, npy_intp dst_stride,
-                        char *src, npy_intp src_stride,
-                        npy_intp N, npy_intp src_itemsize,
-                        NpyAuxData *NPY_UNUSED(data))
+_swap_strided_to_strided(
+        PyArrayMethod_Context *context, char *const *args,
+        const npy_intp *dimensions, const npy_intp *strides,
+        NpyAuxData *NPY_UNUSED(data))
 {
+    npy_intp N = dimensions[0];
+    char *src = args[0], *dst = args[1];
+    npy_intp src_stride = strides[0], dst_stride = strides[1];
+    npy_intp src_itemsize = context->descriptors[0]->elsize;
+
```

Comment:
If this is never used maybe we do not need it?

Reply:
ugggh, it might be used for unaligned, byte-swapped unicode strings or something, let me try to cover it...

Reply:
No, nevermind... this is impossible to hit.  It *might* have been possible to hit with user dtypes, because I won't rule out NumPy assumed it knows how to byte-swap them.  This code doesn't actually byte-swap unicde correctly (I tried, but its obvious, since it swaps the whole string rather than each element.)

I reluctant to delete it, since it will be hit if a compiler would actually map `long long` to an `int128` though.

Reply:
Actually would need int256, so if  you prefer, we could just make the branch that would end up using this an error instead.

Reply:
I would be fine with a comment about where it is useful